src/ContentMessages.tsx

```diff
@@ -334,6 +334,7 @@ function uploadFile(matrixClient: MatrixClient, roomId: string, file: File | Blo
             if (file.type) {
                 encryptInfo.mimetype = file.type;
             }
+            // TODO: Blurhash for encrypted media?
```

Comment:
To be resolved by the MSC before this PR, unless this PR takes a while to be reviewed in which case this magically disappears.

Reply:
If you look at the [spec](https://matrix.org/docs/spec/client_server/latest#extensions-to-m-message-msgtypes), encryption just replaces `url` with `file` (same for thumbnails). All the other properties in `content` still get encrypted (apart from `relates_to`), so for encrypted media the blurhash should already be present and get encrypted. That's also how clients currently handle it (Nheko).

Reply:
having looked at how we could support this in practice, I'm inclined to leave encrypted media out of this PR entirely due to complexity.